packages/pegasus/src/proposals/core-proposal.js

```diff
@@ -104,7 +104,7 @@ export const listenPegasus = async ({
   pegasusConnectionsAdmin.resolve(nameAdmin);
 
   const pegasus = await E(zoe).getPublicFacet(pegasusInstance);
-  const port = await E(networkVat).bindPort('/ibc-port/pegasus');
+  const port = await E(portAllocator).allocateIBCPegasusPort();
```

Comment:
This makes me somewhat uncomfortable.  It seems to illustrate that `allocateIBCPegasusPort()` is too specific and doesn't really solve the need for custom port names.  Is there a solution that exists between the power of `bindPort()` and the rigidity of `allocateIBCPegasusPort()`?